### DISCUSSION TITLE
truffle unbox react - Unbox Failed (Windows 10)

### DISCUSSION THREAD

**Discussion by dmm22** [ConsenSys-archive/truffle#4011] [General Q&A]
Wait, I Figured it out! Sorry!

**Comment by BlasterZ32** [comment]
What was the problem and how did you figure it out?

**Reply by apdan510** [reply]
Blaster are you having issues with the react boxes?

**Comment by BlasterZ32** [comment]
Hey, I haven’t been through the course for a while. I highly recommend
crypto zombies to learn solidity though
